=== modified file 'django/core/management/commands/syncdb.py'
--- django/core/management/commands/syncdb.py	2008-06-19 13:24:39 +0000
+++ django/core/management/commands/syncdb.py	2008-06-27 23:37:55 +0000
@@ -102,10 +102,6 @@
 
         transaction.commit_unless_managed()
 
-        # Send the post_syncdb signal, so individual apps can do whatever they need
-        # to do at this point.
-        emit_post_sync_signal(created_models, verbosity, interactive)
-
         # Install custom SQL for the app (but only if this
         # is a model we've just created)
         for app in models.get_apps():
@@ -153,3 +149,7 @@
         # Install the 'initial_data' fixture, using format discovery
         from django.core.management import call_command
         call_command('loaddata', 'initial_data', verbosity=verbosity)
+
+        # Send the post_syncdb signal, so individual apps can do whatever they need
+        # to do at this point.
+        emit_post_sync_signal(created_models, verbosity, interactive)

